### PR TITLE
Represent char

### PR DESCRIPTION
**Pull request by lxgr-linux**
Added `represent_char` option to savefile so the player is able to change the their representation on the map from an `a` to something else.

Idea from [here](https://www.reddit.com/r/unixporn/comments/tk9024/comment/i1p9g3o/?utm_source=share&utm_medium=web2x&context=3)

### CODE REVIEW

**Comment by MaFeLP** [comment]
Three things:

1. You should check for the length of the `represent_char` string, as having two characters or more, breaks the whole map behavior (you can not see the first line anymore and all the map downwards on shifts)
2. Exclude Unicode characters from settable characters, as they require sometimes a bit more space in a monospace font, than normal characters (test by setting `represent_char` to `😎` for example
3. Maybe add a dialogue box to the menu, that lets a player change their char easily, as this is a very nice feature!

**Comment by lxgr-linux** [comment]
> Three things:
> 
> 1. You should check for the length of the `represent_char` string, as having two characters or more, breaks the whole map behavior (you can not see the first line anymore and all the map downwards on shifts)
> 2. Exclude Unicode characters from settable characters, as they require sometimes a bit more space in a monospace font, than normal characters (test by setting `represent_char` to `😎` for example
> 3. Maybe add a dialogue box to the menu, that lets a player change their char easily, as this is a very nice feature!

Done, back open for review.

**Review by MaFeLP — CHANGES_REQUESTED** [review]
Please add a Notification or some other indicator that the represent_char is invalid, as the user may not know, why their save file is not working and they are not using the `--log` option.

**Comment by lxgr-linux** [comment]
> Please add a Notification or some other indicator that the represent_char is invalid, as the user may not know, why their save file is not working and they are not using the `--log` option.

Done

File: pokete.py [hunk 1]
```diff
@@ -523,25 +531,38 @@ def __init__(self, _map):
...
                                                  figure.name, 18, 17)
                         self.map.name_label_rechar(figure.name)
+                    elif i == self.represent_char_label:
+                        inp = text_input(self.char_label, self.map,
+                                         figure.char, 18, 1)
+                        # excludes bad unicode:
+                        if len(inp.encode("utf-8")) != 1:
+                            inp = "a"
+                            notifier.notify("Error", "Bad character",
+                                            "The choosen chracater has to be a \
```
Comment (MaFeLP) [inline pokete.py:563]: Fix typo

```suggestion
                                            "The chosen character has to be a \
```

**Review by MaFeLP — CHANGES_REQUESTED** [review]
Apart from these two typos, this pull request looks good for merging!

**Pull request merged**